report/assets/faux-psi-template.html

```diff
@@ -0,0 +1,120 @@
+<!--
```

Comment:
I feel like these two files could live somewhere else. or at least renamed folder to `test-assets` to make it clear they aren't actually runtime assets of the report. or moved to `report/test/assets`.